README.md

```diff
@@ -62,7 +62,14 @@ java -jar target/mod-orders-storage-fat.jar loadSample=true
 
 Note: The priority for the parameters in the order Tenant Parameters > Command Line argument > Default value
 
-
+### Search and Filtering on APIs
+For the APIs below separate views are created to enable queries that contain fields from Composite Purchase Order.
+* /orders
```

Comment:
Just looking at this change I realized that this endpoint has to be changed to `/orders-storage/orders` to align with [Acquisitions API Listing](https://docs.google.com/spreadsheets/d/1se-a2owRfHLG5eaAZglx4vLvRClCKGp2wZCfD39ZJRY/view#gid=894044560)
@khandramai please prepare PR